lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -835,7 +864,7 @@ export default class BitcoinProcessor {
     const requestString = JSON.stringify(request);
     // console.debug(`Fetching ${fullPath}`);
     // console.debug(requestString);
-    console.debug(`Sending jRPC request: id: ${request.id}, method: ${request['method']}`);
+    console.debug(`Sending jRPC request: ${JSON.stringify(request)}`);
```

Comment:
I'd prefer some smarter logic here so we don't end up logging things that would never really be useful like the signed raw transaction on broadcast, but at the very least could this be:
```suggestion
    console.debug(`Sending jRPC request: ${requestString}`);
```

Reply:
Reverting the log message back to the previous one.